src/main/resources/META-INF/hudson.remoting.ClassFilter

```diff
@@ -1,2 +0,0 @@
-# For backward compatibility with 1.4.2-
-java.util.RandomAccessSubList
```

Comment:
The sublist class has moved in Java 11. Since recent Jenkins core versions require Java 11, this implies we can no longer support upgrade from 1.4.2 which saved data with the Java 8 class name. As such, bumping the compatible since version to 1.5.0 and removing any migration logic.